src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlFieldMonacoEditorSettings.Edit.cshtml

```diff
@@ -1,7 +1,8 @@
+@using OrchardCore.Mvc.Core.Utilities
 @model OrchardCore.ContentFields.ViewModels.MonacoSettingsViewModel
 
 @{
-    var IStandaloneEditorConstructionOptionsSchemaPath = Url.Content("~/OrchardCore.Resources/Scripts/monaco/IStandaloneEditorConstructionOptions.json");
+    var IStandaloneEditorConstructionOptionsSchemaPath = Url.ToAbsoluteUrl("~/OrchardCore.Resources/Scripts/monaco/IStandaloneEditorConstructionOptions.json");
```

Comment:
Add a comment to all of these why this unusual absolute URL approach is necessary.

Reply:
Ok, comments added:
`// Use Url.ToAbsoluteUrl() because Url.Content() leads to a js error: 'Not allowed to load local resource'.`